docs/DeveloperGuide.md

```diff
@@ -153,6 +153,26 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Adding Items and Delivery
+OneShelf is capable of adding items and deliveries.
+Adding Items and Delivery both are done similarly which will be illustrated with an AddItemActivityDiagram below.
+
+![AddItemActivityDiagram](images/AddItemActivityDiagram.png)
+
+Apart from adding a new Item, should there be an existing item in the Inventory Book, `Add` will be able to increase the quantity of that particular Item.
+In this case 2 items are considered the same if they have the same `Name` and `Supplier`.
+The other variables such as:
+
+a) `Metric` `MaxQuantity`
+- Are not allowed to be defined if there exists the same item inside InventoryBook.
+
+b) `Tags`
+- Will be combined together if there exist the same item inside InventoryBook.
+
+Note: Deliveries are all considered unique. Reason being the same person can make multiple delivery orders.
+
+For example, `John` with the address `Choa Chu Kang Block 259` is able to make multiple orders before his previous deliveries are fulfilled.
```

Comment:
U can use the information source for this kind of note to make it consistent

Reply:
Added! Thanks!